x-pack/plugins/observability_solution/apm/common/service_health_status.ts

```diff
@@ -34,29 +34,35 @@ export function getServiceHealthStatus({ severity }: { severity: ML_ANOMALY_SEVE
   }
 }
 
-export function getServiceHealthStatusColor(theme: EuiTheme, status: ServiceHealthStatus) {
+export function getServiceHealthStatusColor(
+  euiTheme: EuiThemeComputed,
+  status: ServiceHealthStatus
+) {
   switch (status) {
     case ServiceHealthStatus.healthy:
-      return theme.eui.euiColorVis0;
+      return euiTheme.colors.vis.euiColorVis0;
```

Comment:
Fyi, these vis colors changed, they won't map to the same type of colors anymore. That's generally unwanted for severity colors. (see this [related conversation](https://github.com/elastic/kibana/pull/204352#discussion_r1886896464))

We're [introducing new severity color](https://github.com/elastic/eui/pull/8247)s to the theme, which should cover this. But it won't block this PR, the update can be done as follow-up.